docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<ul><li>I'd use weakly informative priors for the Dirichlet, as you do for the other distributions. In his book, Osvaldo recommends that the <code>a</code>  vector be between 4 and 10.</li><li>You can get rid of the <code>FutureWarnings</code>  by doing:</li></ul><pre class="ql-syntax" spellcheck="false">import warnings

warnings.simplefilter(action="ignore", category=FutureWarning)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>